images/kindnetd/cmd/kindnetd/cni.go

```diff
@@ -34,19 +36,41 @@ import (
 type CNIConfigInputs struct {
 	PodCIDR      string
 	DefaultRoute string
+	Mtu          int
 }
 
 // ComputeCNIConfigInputs computes the template inputs for CNIConfigWriter
 func ComputeCNIConfigInputs(node corev1.Node) CNIConfigInputs {
 	podCIDR := node.Spec.PodCIDR
 	defaultRoute := "0.0.0.0/0"
+	mtu, err := computeBridgeMTU()
```

Comment:
@BenTheElder if I read it correctly this is going to be executed periodically, can you confirm?

I don't like the idea of iterating over the interfaces periodically, it can cause issues, I had issue with the net library before.
Should be better to try to get the mtu only once? in the main loop? and pass it as a parameter of ComputeCNIConfigInputs?

Reply:
Actually, yes it will be computed multiple times and I think it's a more sound approach to compute this once.